docs/design/user-memory.md

```
@@ -132,11 +132,12 @@ User Action (send message, save file, search, save note)
   → Available for search immediately
 ```
 
-The pipeline is async — never blocks the user action. Uses the same QMD embed API as agent memory:
+The pipeline is async — never blocks the user action. Writes reach taosmd via:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)